frontend/src/pages/HomePage.tsx

```
@@ -37,24 +95,53 @@ const HomePage = () => {
+          {!(isLoading || hasTrips) &&
+            (isArchivedView ? (
+              <p className='mt-8 text-center text-gray-500'>アーカイブした旅程はありません。</p>
+            ) : (
+              <div className='relative flex flex-col items-center px-24 sm:px-0'>
+                <p className='relative mt-8 w-full text-center text-gray-500'>
+                  旅程がまだありません。
+                  <br />
+                  共有してもらうか新しく作りましょう！
+                </p>
+                {/* テキストから右上ボタンへの点線カーブ矢印 */}
+                <CurvedArrow className='-top-6 absolute right-4 h-24 text-gray-400 sm:right-16' />
```

Biome (2.5.6)

[error] These CSS classes should be sorted.

(lint/nursery/useSortedClasses)